ambry-router/src/test/java/com/github/ambry/router/GetManagerTest.java

```diff
@@ -282,12 +282,16 @@ private String createStitchBlob(int blobSize) throws Exception {
     int curBlobSize = blobSize;
     for (int i = 0; i < NUM_STITCHED_CHUNKS; i++) {
       //Give each chunk a different size
-      int curChunkSize = Math.min(curBlobSize, chunkSize + i * 5);
+      int curChunkSize = Math.min(curBlobSize, chunkSize - i * 5);
+      if (i == NUM_STITCHED_CHUNKS - 1) {
+        curChunkSize = curBlobSize;
+      }
```

Comment:
Why we change logic here?

Reply:
Because before this change, both simple blobs and chunked uploads were marked as `DATACHUNKS`. So it was totally possible to stitch `SIMPLE` blobs. Going forward, `SIMPLE` blobs will not be allowed for stitching. So, in order to upload blobs that need to be stitched later (like this test does), we need to upload it with option `chunkUpload=true`. 
Now once the blob is marked as chunked upload, then we also check and ensure that the max upload size doesn't exceed `CHUNK_SIZE`. Hence it's important to calculate the chunk size correctly.